tool/common/common.go

```diff
@@ -116,11 +116,11 @@ type ClusterAlertGetter interface {
 }
 
 // ShowClusterAlerts shows cluster alerts with the given labels and severity.
```

Comment:
So we know show everything at the specified severity and higher? If so, can you update the godoc here? As written it sounds like we only show exactly the severity you asked for.